platforms/r3-corda-ent/charts/enterprise-node/files/node.conf

```diff
@@ -38,7 +38,7 @@ jmxMonitoringHttpPort: {{ .Values.nodeConf.monitoring.port }}
 trustStorePassword: {{ .Values.nodeConf.creds.truststore }}
 keyStorePassword : {{ .Values.nodeConf.creds.keystore }}
 detectPublicIp: false
-additionalP2PAddresses: []
+additionalP2PAddresses: ["{{ .Release.Name }}.{{ .Values.global.proxy.externalUrlSuffix }}:443"]
```

Comment:
Are we using 443 here or any other port like 15011?

Reply:
P2P use TCP protocol, the HTTPS ambassador mapping seems to work (from the logs). Have not tested the cordapps, so not sure if the p2p really works. We might have to switch to tcpmapping with ports like 15011, if we face issues.